backend/dataall/cdkproxy/stacks/dataset.py

```diff
@@ -389,53 +309,94 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
                 iam.AccountPrincipal(os.environ.get('CURRENT_AWS_ACCOUNT')),
                 iam.AccountPrincipal(dataset.AwsAccountId),
                 iam.ArnPrincipal(
-                    f'arn:aws:iam::{dataset.AwsAccountId}:role/{pivot_role_name}'
+                    f'arn:aws:iam::{dataset.AwsAccountId}:role/{self.pivot_role_name}'
                 ),
             ),
         )
         dataset_admin_policy.attach_to_role(dataset_admin_role)
 
-        glue_db_handler_arn = ssm.StringParameter.from_string_parameter_name(
+        # Datalake location custom resource: registers the S3 location in LakeFormation
```

Comment:
I would be happy if we remove it to keep the code clean, we can always find it in the history once the bug is fixed and we need to revert it back